### ISSUE TITLE
Failed to update score database.

### ISSUE THREAD

**Issue by BeDoXGamer** [fluddokt/opsu#31]
Failed to update score database.
OpsuAnd0.12.1a
java.sql.SQLException: android.database.sqlite.SQLiteException: duplicate column name: replay (code 1): , while compiling: ALTER TABLE scores ADD COLUMN replay TEXT
###### 

Error Code : 1 (SQLITE_ERROR)
Caused By : SQL(query) error or missing database.
    (duplicate column name: replay (code 1): , while compiling: ALTER TABLE scores ADD COLUMN replay TEXT)
###### 

```
at java.lang.reflect.Constructor.newInstance(Native Method)
at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
at org.sqldroid.SQLDroidConnection.chainException(SQLDroidConnection.java:158)
at org.sqldroid.SQLiteDatabase.execSQL(SQLiteDatabase.java:170)
at org.sqldroid.SQLDroidStatement.executeUpdate(SQLDroidStatement.java:173)
at itdelatrisu.opsu.db.ScoreDB.updateDatabase(ScoreDB.java:208)
at itdelatrisu.opsu.db.ScoreDB.init(ScoreDB.java:95)
at itdelatrisu.opsu.db.DBController.init(DBController.java:50)
at itdelatrisu.opsu.Opsu.main(Opsu.java:188)
at fluddokt.opsu.fake.GameOpsu.render(GameOpsu.java:92)
at com.badlogic.gdx.backends.android.AndroidGraphics.onDrawFrame(AndroidGraphics.java:422)
at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1561)
at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1278)
```

Caused by: android.database.sqlite.SQLiteException: duplicate column name: replay (code 1): , while compiling: ALTER TABLE scores ADD COLUMN replay TEXT
###### 

Error Code : 1 (SQLITE_ERROR)
Caused By : SQL(query) error or missing database.
    (duplicate column name: replay (code 1): , while compiling: ALTER TABLE scores ADD COLUMN replay TEXT)
###### 

```
at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:1093)
at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:670)
at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:59)
at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
at android.database.sqlite.SQLiteDatabase.executeSql(SQLiteDatabase.java:1812)
at android.database.sqlite.SQLiteDatabase.execSQL(SQLiteDatabase.java:1743)
at org.sqldroid.SQLiteDatabase.execSQL(SQLiteDatabase.java:163)
... 9 more
```

**Comment by BeDoXGamer** [comment]
What my problem??

**Comment by fluddokt** [comment]
Should be fixed in 0.15.0b
